hcl_dx_jsf_demoportlet_archetype/src/main/resources/archetype-resources/src/main/java/pagecode/PageCodeBase.java

```diff
@@ -0,0 +1,229 @@
+package pagecode;
```

Comment:
Missing copyright in java files - see e.g. https://github.com/HCL-TECH-SOFTWARE/dx-portlet-development-utilities/blob/main/pageimport/src/main/java/com/hcl/dx/sample/pages/MyPortlet.java#L1

Reply:
I added the copyright to the java files